mmv1/templates/terraform/examples/compute_interconnect_basic.tf.erb

```diff
@@ -2,7 +2,6 @@ data "google_project" "project" {}
 
```

Comment:
Looks like this example is skipped from test generation. It will update the examples in the web documentation though. Would you mind also removing the field from [this example](https://github.com/abhijeetkjha-google/magic-modules/blob/bugFix-361094483/mmv1/templates/terraform/examples/compute_interconnect_basic_test.tf.erb) to have a test coverage on this. Plus, can you also remove this line https://github.com/abhijeetkjha-google/magic-modules/blob/bugFix-361094483/mmv1/products/compute/Interconnect.yaml#L52, given the field is removed from the example. Thanks

Reply:
> Would you mind also removing the field from [this example](https://github.com/abhijeetkjha-google/magic-modules/blob/bugFix-361094483/mmv1/templates/terraform/examples/compute_interconnect_basic_test.tf.erb) to have a test coverage on this. 

Since we are marking a field as optional wouldn't it be a good idea to have one test example with the field and another without the field. The compute_interconnect_basic.tf.erb contains the required field only and compute_interconnect_basic_test.tf.erb includes the optional field as well.

> can you also remove this line https://github.com/abhijeetkjha-google/magic-modules/blob/bugFix-361094483/mmv1/products/compute/Interconnect.yaml#L52, given the field is removed from the example.

Sure, Done.

Reply:
Yes yes, I have verified that customer_name is an optional field and is not required for resource creation.